configuration.go

```diff
@@ -272,7 +272,8 @@ func NewTraefikDefaultPointersConfiguration() *TraefikConfiguration {
 	//default Kubernetes
 	var defaultKubernetes provider.Kubernetes
 	defaultKubernetes.Watch = true
-	defaultKubernetes.Endpoint = "http://127.0.0.1:8080"
+	defaultKubernetes.Endpoint = ""
```

Comment:
To know if it was explicitly provided to decide whether to use the
kubernetes provide environment variables. It falls back to the 127 default
otherwise

On Wednesday, July 13, 2016, Emile Vauge <notifications@github.com> wrote:

> In configuration.go
> <https://github.com/containous/traefik/pull/516#discussion_r70667451>:
>
> > @@ -272,7 +272,8 @@ func NewTraefikDefaultPointersConfiguration() *TraefikConfiguration {
> >  	//default Kubernetes
> >  	var defaultKubernetes provider.Kubernetes
> >  	defaultKubernetes.Watch = true
> > -	defaultKubernetes.Endpoint = "http://127.0.0.1:8080"
> > +	defaultKubernetes.Endpoint = ""
>
> Why did you remove this default value ?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/containous/traefik/pull/516/files/9f6484a328127189009d83dd2758115ccc4529c1#r70667451>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAy2K8s6E9x8sQMKsWvvaLn_IL666OMEks5qVRyKgaJpZM4JKDPu>
> .
>